### ISSUE TITLE
Custom icons not loading up.

### ISSUE THREAD

**Issue by almantas88** [FlutterFlow/flutterflow-issues#1797]
### Has your issue been reported?

- [X] I have searched the existing issues and confirm it has not been reported.
- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

Weird symbol shows up when using a custom icon:
![image](https://github.com/FlutterFlow/flutterflow-issues/assets/47143702/4039bea3-dd5b-4aef-8ccd-a296bf68056b)

### Expected Behavior

That it would show correctly these symbols that I have uploaded:
![image](https://github.com/FlutterFlow/flutterflow-issues/assets/47143702/5a615930-5e66-456e-864b-7dda9bee7fb0)

### Steps to Reproduce

1. Create a project;
2. Import custom icons;
3. Create an Icon widget and select any custom icon;

### Reproducible from Blank

- [X] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

IT4Ok/Lqx8pgod9c7ZPpbcAwmikRHGYlU4ZN0utHbAE3GKnNG7UIec/4VEptXbC+TVc3f1r9oHge+daMv+bIN8ICZUmBUa5FwqtQYAyXLVK6MoSQPpGwXnJ6AtpKClCD17a3hR1TBOxeSUYO2zquPOTKbDDfGO/NOGoZRf2hNrz7s3/uQ0SLb3kNh1JWeC/v

### Context

I'm trying to display a custom icon.

### Visual documentation

![image](https://github.com/FlutterFlow/flutterflow-issues/assets/47143702/7faecc5a-67d4-40c2-b335-65dbb9f64bba)

### Environment

```markdown
- FlutterFlow version: 3.13.7
- Platform:
- Browser name and version:
- Operating system and version affected:
```

**Comment by hariprasadms** [comment]
Hi, Thanks for submitting the issue. I have tested above steps using custom icons generated from [icomoon](https://icomoon.io/) . All works fine as expected. 

<img width="777" alt="image" src="https://github.com/FlutterFlow/flutterflow-issues/assets/17435678/664b43b2-6507-45c0-a6b9-37454a401e8a">


What is the source of custom icons you generated ?
Is this behaviour you are seeing in run mode or on editor ?

Refer this youtube [video](https://www.youtube.com/watch?v=fJ8tCE_767A) to check you have followed the correct steps while generate and uploading icons.

Updated these details in the issues please. Thanks

**Comment by almantas88** [comment]
Im testing on a physical device, Samsung s10e.

It was working befora the latest update.

**Comment by almantas88** [comment]
Any updates?

**Comment by almantas88** [comment]
Sooo any updates?

**Comment by hariprasadms** [comment]
Hi @almantas88 - Sorry for the delayed response on the issue. I have tested it on Android emulator. I can see custom icons are showing up property. See below.

<img width="383" alt="image" src="https://github.com/FlutterFlow/flutterflow-issues/assets/17435678/4201c246-dfe5-4d14-b58a-44b32b072588">

are you still facing the issue ? Yes, add below details 

Source of custom icons ? - I have tested using iconmoon icons.
What are those actual icons you are selecting?
